Description: disable IPv4 only tests
 As these tests are IPv4 only, fail to work on IPv6 hosts. Disable to prevent
 FTBFS.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Bug-Debian: https://bugs.debian.org/1011011
Last-Update: 2022-05-15

---

--- a/lib/cpp/test/Makefile.am
+++ b/lib/cpp/test/Makefile.am
@@ -82,7 +82,6 @@ Benchmark_SOURCES = \
 Benchmark_LDADD = libtestgencpp.la
 
 check_PROGRAMS = \
-	UnitTests \
 	TFDTransportTest \
 	TPipedTransportTest \
 	DebugProtoTest \
@@ -105,9 +104,9 @@ check_PROGRAMS = \
 if AMX_HAVE_LIBEVENT
 noinst_PROGRAMS += \
 	processor_test
-check_PROGRAMS += \
-	TNonblockingServerTest \
-	TNonblockingSSLServerTest
+#check_PROGRAMS += \
+#	TNonblockingServerTest \
+#	TNonblockingSSLServerTest
 endif
 
 TESTS_ENVIRONMENT= \
